server/service/src/invoice/inbound_shipment/insert/validate.rs

```diff
@@ -27,18 +30,3 @@ pub fn validate(
 
     Ok(other_party)
 }
-
-fn check_invoice_does_not_exists(
```

Comment:
Moved to common validation since used in three places now and refactored a bit...